int64 x
int64 y
int64 w
int64 h
string color
string clase
float32 X
float32 Y
